testing/integration/Cargo.toml

```diff
@@ -23,6 +23,6 @@ serde = { workspace = true }
 serde_tuple = { workspace = true }
 
 [dev-dependencies]
-actors-v12 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors", branch = "chore/update-fvm" }
+actors-v12 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors", branch = "fvm-next" }
```

Comment:
This will switch back to master after merging https://github.com/filecoin-project/builtin-actors/pull/1419.

Reply:
And before you release libraries?